memote-report-app/src/app/report-data.service.ts

```diff
@@ -39,7 +39,7 @@ export class ReportDataService {
   public isScored(string) {
     return this.scoredTests.includes(string);
   }
-
+  
```

Comment:
No need to add whitespace here, you might want to configure your editor/IDE to automatically trim trailing whitespace